src/ParkChecklist/ParkChecklist.js

```diff
@@ -123,161 +93,119 @@ const ParkChecklist = ({ apiLink }) => {
-                    )
-                )
-                .map((park) => (
-                  <li key={park.id}>
-                    <input
-                      type="checkbox"
-                      id={park.id}
-                      checked={checkedItems.includes(park.id)}
-                      onChange={() => handleCheckboxChange(park.id)}
-                    />
-                    <label htmlFor={park.id}>{park.attributes.name}</label>
-                  </li>
-                ))
-            ) : selectedFilters.designations.length > 0 ? (
-              parks
-                .filter((park) =>
-                  selectedFilters.designations.includes(
-                    park.attributes.designation
-                  )
-                )
-                .map((park) => (
-                  <li key={park.id}>
-                    <input
-                      type="checkbox"
-                      id={park.id}
-                      checked={checkedItems.includes(park.id)}
-                      onChange={() => handleCheckboxChange(park.id)}
-                    />
-                    <label htmlFor={park.id}>{park.attributes.name}</label>
-                  </li>
-                ))
-            ) : (
-              currentParks.map((park) => (
-                <li key={park.id}>
-                  <input
-                    type="checkbox"
-                    id={park.id}
-                    checked={checkedItems.includes(park.id)}
-                    onChange={() => handleCheckboxChange(park.id)}
-                  />
-                  <label htmlFor={park.id}>{park.attributes.name}</label>
-                </li>
-              ))
-            )}
-          </ul>
-          <div className="pagination">
-            {currentPage !== 1 && (
-              <button onClick={() => handlePageChange(1)}>{"<<"}</button>
-            )}
-            {currentPage > 1 && (
-              <button onClick={() => handlePageChange(currentPage - 1)}>
-                {"<"}
-              </button>
-            )}
-            <span>{`${currentPage}/${Math.ceil(
-              parks.length / parksPerPage
-            )}`}</span>
-            {currentPage < Math.ceil(parks.length / parksPerPage) && (
-              <button onClick={() => handlePageChange(currentPage + 1)}>
-                {">"}
-              </button>
-            )}
-            {currentPage !== Math.ceil(parks.length / parksPerPage) && (
-              <button
-                onClick={() =>
-                  handlePageChange(Math.ceil(parks.length / parksPerPage))
-                }
-              >
-                {">>"}
-              </button>
-            )}
-          </div>
-        </>
-      ),
-    },
-    {
-      path: `/login`,
-      element: (
-        <div>
-          <Login loggedIn={loggedIn} setLoggedIn={setLoggedIn} />
+  return (
+    <div className="park-checklist-container">
+      {loading ?? <p className="loading-message">Loading Parks...</p>}
```

Comment:
Ah I did miss the loading message on the first go around. Once we have this merged, I can check the useState that it is conditionally connected to and make sure it's being changed at the right time.

Reply:
Loading has been fixed with Lex's help!